webapp/app/forms/flows/step_chooser.py

```diff
@@ -42,26 +40,29 @@ def _get_possible_redirect(self, step_name):
         else:
             return None
 
-    def get_correct_step(self, step_name) -> SteuerlotseStep:
+    def get_correct_step(self, step_name: str, update_data: bool) -> SteuerlotseStep:
```

Comment:
Maybe default to `False` then the tests don't need to change as much?

```suggestion
    def get_correct_step(self, step_name: str, update_data: bool = False) -> SteuerlotseStep:
```

Reply:
I first had it like that...if I'd only remember the reason I changed it.. 🙈